config/initializers/config_check.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Check key settings in production to make sure they have safe and sensible values.
+
+# rubocop:disable Style/IfUnlessModifier
+# Don't abort if we're running via a rake task b/c we need to be able to run migrations on deploy
+# since migrations will copy the config to .env.
+if Rails.env.production? && File.split($0).last != "rake"
+  secret_key_base = Rails.configuration.secret_key_base
+  if secret_key_base.blank? || secret_key_base.match?(/XXXXXXXXXXXXXXXX/)
+    abort("NEMO_SECRET_KEY_BASE must be set to a real value")
```

Comment:
👏 👏 👏 👏 👏 👏 👏 👏 👏 👏 👏 👏 👏

Reply:
Oh, fyi more flexible regex: `/(.)\1{8}/` to detect any obscenely repeated characters. Maybe we could just use this as a constant everywhere.

[Capture any `.`, refer to it with `\1`, then check if it happens 8 times.]

Reply:
The main thing I'm trying to catch here is if folks haven't overridden the `.env` file values, which we control. So I don't think we need this extra complexity. I would also have to test it, and testing this stuff is a pain. So I'd like to leave as is for now.